src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -336,8 +336,9 @@ public CompletableFuture<InitializeResult> initialize(InitializeParams params) {
         setNotebookSyncOptions(c);
       }
 
+      var eslintBridgeServerPath = (String) additionalAttributes.get("eslintBridgeServerPath");
```

Comment:
IMHO, this field should be sent in the "root" options object. The `additionalAttributes` field is passed to the telemetry (and I think it may need a bit of cleanup and/or be renamed).